shell/platform/tizen/BUILD.gn

```diff
@@ -121,6 +122,20 @@ template("embedder_for_profile") {
       "wayland-client",
     ]
 
+    if (target_name == "mobile") {
+      libs += [
+        "capi-base-common",
+        "feedback"
+      ]
+    }
+
+    if (target_name == "wearable") {
+      libs += [
+        "capi-base-common",
+        "feedback"
+      ]
+    }
```

Comment:
is there any particular reason of using two separated if's with the same body? Why not to use one if with '||' ?

If this is a design to keep separated sections for mobile and wearable libs for future, then it is ok to me.

Reply:
> If this is a design to keep separated sections for mobile and wearable libs for future, then it is ok to me.

Yes, that's why I separated them